src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -29,13 +29,42 @@
             <div class="row" v-if="workflowRunStatus" id="workflow-table">
                 <label class="form-label col">Reports</label>
                 <table class="table-bordered col-10">
-                    <tr v-for="report in workflowRunStatus.reports">
+                    <tr>
+                        <th class="p-2">Report</th>
+                        <th v-if="anyParams" class="p-2">Parameters</th>
+                        <th class="p-2">Status</th>
+                        <th class="p-2">Logs</th>
+                    </tr>
+                    <tr v-for="(report, index) in workflowRunStatus.reports" :key="report.key">
                         <td v-if="report.status === 'success'" class="p-2">
                             <a class="report-version-link" :href="reportVersionHref(report.name, report.version)">
                                 {{ report.name }}
                             </a>
                         </td>
                         <td v-else class="p-2">{{ report.name }}</td>
+                        <td v-if="anyParams" class="p-2">
+                            <div v-if="hasParams(report)">
+                                <p class="non-default-param"
+                                    v-for="param in report.param_list"
+                                    :key="param.name">{{ param.name }}: {{ param.value }}</p>
+                                <p v-if="!report.param_list.length">No non-default params</p>
+                                <div v-if="report.default_param_list.length > 0"
+                                        :id="`default-params-${index}`">
+                                    <b-link href="#"
+                                            class="show-defaults pt-2 d-inline-block small"
+                                            v-b-toggle="`collapseSummary-${index}`">
+                                        <span class="when-closed">Show</span>
+                                        <span class="when-open">Hide</span> defaults...
+                                    </b-link>
+                                    <b-collapse :id="`collapseSummary-${index}`">
+                                        <p :id="`default-params-collapse-${index}-${paramIndex}`"
+                                            v-for="(param, paramIndex) in report.default_param_list"
+                                            :key="param.name">{{ param.name }}: {{ param.value }}</p>
+                                    </b-collapse>
+                                </div>
+                            </div>
```

Comment:
This repeats logic from `workflowSummaryReports` with a few minor adjustments, such as the "No non-default params" paragraph. Could potential have both shave a component instead?

Reply:
Agreed - if logic is largely shared, better to put that in shared component. I actually think it would be better for the Workflow summary to have the 'No non-default parameters' message too as we should really be consistent.  So probably don't even need props to switch behaviour in the shared component. 

Also - can we not use 'params' in text displayed to user - I think we should probably always say 'parameters'.